examples/catalyst_contrastive/supervised_contrastive.py

```diff
@@ -0,0 +1,137 @@
+# flake8: noqa
+import argparse
+
+import torch
+import torch.nn.functional as F
+from torch.optim import Adam
+import torchvision
+
+from catalyst import dl
+from catalyst.contrib import nn
+from catalyst.contrib.models.cv.encoders import ResnetEncoder
+from catalyst.contrib.nn.criterion import NTXentLoss
+from catalyst.contrib.nn.criterion.supervised_contrastive import SupervisedContrastiveLoss
+from catalyst.data.dataset.self_supervised import SelfSupervisedDatasetWrapper
```

Comment:
could we use it as `data.SelfSupervisedDatasetWrapper`?

Reply:
[Done](https://github.com/catalyst-team/catalyst/pull/1293/commits/57766500548d68bbf3c7dcaaa9db0525fa3e0179)